install_config/configuring_azure.adoc

```diff
@@ -11,26 +11,45 @@
 toc::[]
 
 == Overview
-{product-title} can be configured to access an
-link:https://azure.microsoft.com/en-us/services/storage/disks/[Azure
+{product-title} can be configured to access
+link:https://azure.microsoft.com/en-us/services/storage/disks/[Microsoft Azure
 infrastructure], including
 xref:../install_config/persistent_storage/persistent_storage_azure.adoc#install-config-persistent-storage-persistent-storage-azure[using
-Azure disk as persistent storage] for application data. After Azure is
+Azure disk as persistent storage] for application data. After Microsoft Azure is
 configured properly, some additional configurations need to be completed on the
 {product-title} hosts.
 
 [[configuring-azure-permissions]]
 == Permissions
-Configuring Azure for {product-title} requires the following role:
+Configuring Microsoft Azure for {product-title} requires the following role:
 
 [cols='1,3']
 |===
 
 | Contributor
-| To create and manage all types of Azure resources.
+| To create and manage all types of Microsoft Azure resources.
 
 |===
 
+[[configuring-azure-prerequisites]]
+== Prerequisites
+
+* If you are using Microsoft Azure Disk as an OpenShift Persistent Volume for  {product-title}'s latest version (or lower upto version 3.5), you must enable Azure Cloud Provider.
```

Comment:
I think I'm confused by the above sentence. Maybe:

"If you are using Microsoft Azure Disk as a persistent volume on the {product-title} version 3.5 or higher, you must enable Azure Cloud Provider."

That said, you use "Azure Disk" above without a "Microsoft" preceding it. So I'm not sure what the standard is? Also, does persistent volume need caps and "OpenShift" before it? Also, not sure what "lower upto version 3.5). Hence the above.

Reply:
EDIT: IBM says use later and not higher for version numbers.

Reply:
Thanks I will update it to `If you are using Microsoft Azure Disk as a persistent volume on the {product-title} version 3.5 or later, you must enable Azure Cloud Provider.` 

Also `you use "Azure Disk" above without a "Microsoft" preceding it.` nope it says `Microsoft Azure Disk` 
Since this is the first instance I am using `Microsoft Azure Disk` and `Azure Disk` for rest of the references.